### PR TITLE
dist/redhat: install specified version of scylla-enterprise-conf on meta package

### PR DESCRIPTION
**Pull request by syuu1228**
To install specified version of scylla-conf package, we need to add it on Requires.

See scylladb/scylla-enterprise#1203

### CODE REVIEW

**Comment by syuu1228** [comment]
ping

File: dist/redhat/scylla.spec.mustache [hunk 1]
```diff
@@ -7,7 +7,7 @@ Group:          Applications/Databases
 License:        AGPLv3
```
Comment (avikivity) [inline dist/redhat/scylla.spec.mustache]: Please file an issue in the scylla.git repo and reference it instead of the enterprise issue.

Comment (syuu1228) [reply dist/redhat/scylla.spec.mustache]: Created issue #5639, pushed new commit message https://github.com/scylladb/scylla/pull/5599/commits/e689183e54775e671950c750341bf4b60915e87f

**Comment by avikivity** [comment]
Queued as 83092393701ce41c7e0829c3ef9b0ac6f5374472

**Comment by avikivity** [comment]
Dequeued due to conflicts with the dequeing of #5605 , please respin.

**Comment by syuu1228** [comment]
@avikivity rebased with latest next.

**Pull request merged**